src/init.cpp

```diff
@@ -1577,6 +1577,9 @@ bool AppInitMain(NodeContext& node, interfaces::BlockAndHeaderTipInfo* tip_info)
         node.indexers.insert(g_coin_stats_index.get());
     }
 
+    // Init all indexers
+    for (auto indexer : node.indexers) indexer->Init();
```

Comment:
In commit "refactor: index, decouple 'Init' from 'Start'" (26bd60eafc42484d80d5dfef0fa0abb77e7817ce)

Init return value is ignored. I think this needs to return false if it fails. Would also be good to mark Init declaration with `[[nodiscard]]`

Reply:
good catch 👌🏼 .